src/test/java/lab/SeerBatchGenerator.java

```diff
@@ -60,10 +60,14 @@ public String getState() {
     private static final boolean _ALL_REGISTRIES = false;
     private static final SeerRegistry _SINGLE_REGISTRY = SeerRegistry.CT;    // ignored if _ALL_REGISTRIES set to true
     private static final int _NUM_FILES_PER_REG = 1;
-    private static final boolean _ZIP_FILE = true;
+    //private static final boolean _ZIP_FILE = true;
+    private static final boolean _ZIP_FILE = false;
     private static final boolean _APPEND_SIZE = true;
-    private static final NaaccrLayout _LAYOUT = (NaaccrLayout)LayoutFactory.getLayout(LayoutFactory.LAYOUT_ID_NAACCR_15_ABSTRACT);
-    private static final String _FILE_DESTINATION = "./build/";
+    //private static final NaaccrLayout _LAYOUT = (NaaccrLayout)LayoutFactory.getLayout(LayoutFactory.LAYOUT_ID_NAACCR_15_ABSTRACT);
+    private static final NaaccrXmlLayout _LAYOUT = (NaaccrXmlLayout)LayoutFactory.getLayout(LayoutFactory.LAYOUT_ID_NAACCR_XML_18_ABSTRACT);
+
+    //private static final String _FILE_DESTINATION = "./build/";
+    private static final String _FILE_DESTINATION = "C:\\dev\\test\\";
```

Comment:
Normally I wouldn't care about a lab class like this, but some of the changes you did are really not going to work for other people. For example, a hard-coded "C:\" path is not going to work in all environment. Can you just revert all your changes for this lab class? I think the previous version had better defaults.